Dist/FHSDK.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>FH.SDK</id>
-    <version>1.3.8</version>
+    <version>1.3.9-rc1</version>
```

Comment:
@edewit Since we have https://www.nuget.org/packages/FH.SDK/1.3.9-rc2 released, I think this one should be rc3